Sources/AccountSDKIOSWeb/Simplified Login/View/Sub Views/ContinueButton.swift

```diff
@@ -33,16 +33,53 @@ final class ContinueButton: UIButton {
         return view
     }()
 
+    private lazy var nameTitleLabel: UILabel = {
+        let view = UILabel()
+        view.text = title
+        view.isAccessibilityElement = false
+        view.font = UIFont.preferredFont(forTextStyle: .callout).bold()
+        view.textAlignment = .left
+        view.numberOfLines = 1
+        view.lineBreakMode = .byTruncatingTail
+        view.textColor = .white
+        view.translatesAutoresizingMaskIntoConstraints = false
+
+        return view
+    }()
+    
+    private lazy var combinedTitleView: UIView = {
+        return UIView()
+    }()
```

Comment:
Could we use UIStackView for this instead to be able to skip adding some constraints?

Reply:
Done